sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -509,6 +526,11 @@ object ScalaNativePluginInternal {
     files.filterNot(f => isOtherGC(f.getPath().toString, nativelib))
   }
 
+  private val isWindows: Boolean = {
+    val os = Option(sys props "os.name").getOrElse("")
```

Comment:
is it ok like that?
val os = Option(System.getProperty("os.name")).getOrElse("")